### DISCUSSION TITLE
[Feature] QR Code Generation and Facial Recognition for Photo Management

### DISCUSSION THREAD

**Discussion by C0pywriting** [immich-app/immich#10583] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

We propose a feature enhancement for the photo management solution that integrates QR code generation with facial recognition technology. This feature will allow users to generate QR codes which, when scanned, can be used to associate faces with names or create new profiles if they do not already exist in the system. This will streamline the process of managing and tagging photos with accurate identification.

QR Code Generation:
•	As an administrator, I want to generate unique QR codes for individuals.
•	Each QR code will link to a specific user profile or create a new one if the profile doesn't exist.
if an image is found on which the qrcode is found, the face recognition should be compared with this image. The image can be discarded later

For larger events, the qr codes with the names can be created in advance and at the beginning the persons are photographed with the qr code and at the end all persons are assigned.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I don't think this'll be in scope for Immich. It sounds like a great candidate for a future plugin though.
